README.md

```diff
@@ -51,15 +51,16 @@ Reporting here the real time status of the project development branch sanity:
 | ovirt-imageio |  |
 | ovirt-jboss-modules-maven-plugin |  |
 | ovirt-lldp-labeler |  |
-| ovirt-log-collector |  |
+| [ovirt-log-collector](https://github.com/oVirt/ovirt-log-collector) | [![Copr build status](https://copr.fedorainfracloud.org/coprs/ovirt/ovirt-master-snapshot/package/ovirt-log-collector/status_image/last_build.png)](https://copr.fedorainfracloud.org/coprs/ovirt/ovirt-master-snapshot/package/ovirt-log-collector/)
+ |
```

Comment:
The newline here seems to be breaking the formatting.

Reply:
Thanks! fixed